hbase-protocol-shaded/src/main/protobuf/server/rsgroup/RSGroupAdmin.proto

```diff
@@ -183,4 +191,7 @@ service RSGroupAdminService {
 
   rpc RenameRSGroup(RenameRSGroupRequest)
     returns (RenameRSGroupResponse);
+
```

Comment:
This has been added to master.proto. Why add this here too?

Reply:
Didn't notice RSGroupAdminServiceImpl was deprecated, let me remove it.